app/src/main/java/com/jHerscu/clearskies/data/model/DailyForecast.kt

```diff
@@ -0,0 +1,10 @@
+package com.jHerscu.clearskies.data.model
+
+data class DailyForecast(
+    val dateInMill: Long, // Format w/ SimpleDateFormat("MM/dd/yyyy", Locale.getDefault()).format()
```

Comment:
small typo - I think you meant `dateInMillis`

Reply:
I actually hadn't noticed, but I can see how Millis is clearer